pkgs/development/libraries/libavif/default.nix

```diff
@@ -68,6 +68,9 @@ stdenv.mkDerivation rec {
     GDK_PIXBUF_MODULE_FILE=${gdkPixbufModuleFile} \
     gdk-pixbuf-query-loaders --update-cache
 
+  ''
+  # Cross-compiled gdk-pixbuf doesn't support thumbnailers
+  + lib.optionalString (stdenv.hostPlatform == stdenv.buildPlatform) ''
```

Comment:
I think this should be `buildPlatform.canExecute hostPlatform`.

Reply:
Even though gdk-pixbuf uses `meson.is_cross_build()`?

Reply:
Ah, I see, and you explained in the commit message, that the problem isn't inability to *execute* the thumbnailer, but rather that *the thumbnailer doesn't even get built, so it isn't there.